api/tests/routes/pro/patch_collective_offer_test.py

```diff
@@ -184,6 +185,37 @@ def test_patch_collective_offer_update_legacy_template(self, client):
         assert template.domains == [domain]
         assert template.interventionArea == ["01", "2A"]
 
+    def test_patch_collective_offer_replacing_venue_with_same_offerer(self, client):
+        # Given
+        offerer = offerers_factories.OffererFactory()
+        offerers_factories.UserOffererFactory(
+            user__email="user@example.com",
+            offerer=offerer,
+        )
+        offer = CollectiveOfferFactory(
+            venue__managingOfferer=offerer
+            # offer.venueId= 17442
+        )
+        venue2 = offerers_factories.VenueFactory(
+            managingOfferer=offerer,
+        )
+        CollectiveBookingFactory(
+            collectiveStock__collectiveOffer=offer, collectiveStock__beginningDatetime=datetime(2020, 1, 1)
+        )
```

Comment:
La date de l'événement ne semble pas utilisée dans le test, il ne me semble donc pas nécessaire de la spécifier : 

```suggestion
        CollectiveBookingFactory(collectiveStock__collectiveOffer=offer)
```